db/migrate/20180819144937_add_editor_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddEditorToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :editor, :boolean, default: false
+  end
+end
```

Comment:
All these booleans..... Why not use `enum` for `:role` ?

Reply:
The reasoning with my limited experience what, that it seemed as the simplest solution for our MVP, but I could be wrong.
What is the advantage of enum?

Reply:
There’s a guide about role based authorization in your course materials